packages/orbit-components/src/utils/common/index.js

```diff
@@ -1,2 +1,9 @@
 // @flow
+import { css } from "styled-components";
+
 export const firstToUpper = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
+
+export const defaultFocus = (): any => css`
+  outline: 1px solid Highlight;
```

Comment:
wdyt? maybe default focus should have 2px, just check the difference between default browser outline and that utility ;)

Reply:
Right. I don't know where I got the 1px from 🤯